torch/ao/quantization/_quantize_pt2e.py

```diff
@@ -0,0 +1,205 @@
+import torch
+from torch.fx import GraphModule
+import torch._dynamo as torchdynamo
+from torch.nn.utils.fusion import fuse_conv_bn_weights
+
+from torch.ao.quantization.fx.prepare import (
+    is_activation_post_process_node,
+)
+from .qconfig_mapping import QConfigMapping
+from .backend_config import BackendConfig
+from .fx import prepare
+from .quantize_fx import _convert_to_reference_decomposed_fx
+
+from typing import Tuple, Any, Dict
+from collections import OrderedDict
+import operator
+
+# TODO: longer term, don't need to retrace or parse the string
```

Comment:
chagne TODO to TODO(qihan) so I will find it.

Reply:
OK, I also plan to rebase after Sherlock land his PR: https://github.com/pytorch/pytorch/pull/90803, we may not need this